jira-workflows/recipes/issue-details/recipe.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.1.0
+title: "Analyze details for given issue"
+description: "Check details for issue {{ issue }}."
+
+
+instructions: |
+  - Never create, delete, update or modify an Issue in Jira
+  - fixVersion named rhel-N is mapped in branch named cNs
+
+parameters:
+- key: issue
```

Comment:
can we add the param to the recipe invocation in the Makefile please?

Reply:
Sure, I think it will be much easier to add the params for the new recipe working on the changes @vrothberg did. I will wait for rebasing mine PR over the updated main.